index.html

```diff
@@ -0,0 +1,96 @@
+                <input type="text" placeholder="Search">
+                <i class="fa fa-search"></i>
+                <!-- name -->
+            </div>
+            <i class="fa fa-microphone"></i>
+        </div>
+        <div class="nav-right">
+            <i class="fa fa-search"></i>
+            <i class="fa fa-cloud"></i>
+        </div>
+    </nav>
+
+
+    <div class="play-container">
+        <div class="row">
+            <div class="play-video">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/lJIrF4YjHfQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+            <div class="tags">
+                <a href=""></a>
+            </div>
+            <h3>Best Software Development Channel</h3>
+            <div class="video-desc">
+                <p>200 Views &bull; 5 days ago</p>
+                <div>
+                    <a href=""><i class="fa fa-thumbs-up"></i>10k</a>
+                    <a><i class="fa fa-thumbs-down"></i>20</a>
+                    <a><i class="fa fa-share"></i>Share</a>
+                    <a><i class="fa fa-download"></i>Download</a>     
+                </div>
+            </div>
+            <hr>
+            <div class="publisher">
+                
+            </div>
+            
+        </div>
+            <div class="right-sidebar">
+                <div class="side-video-list">
+                    <a href="" class="small-thumbnail">
+                    <img src="/images/thumbnail.png" alt="side-image">
+                    </a>
+                    <div class="vid-info">
+                        <a href="">Best Web development videos</a>
+                        <p>Easy Tutorial</p>
+                        <p>10k views</p>
+                    </div>
+                </div>
+                <div class="side-video-list">
+                    <a href="" class="small-thumbnail">
+                    <img src="/images/thumbnail.png" alt="side-image">
+                    </a>
+                    <div class="vid-info">
+                        <a href="">Best Web development videos</a>
+                        <p>Easy Tutorial</p>
+                        <p>10k views</p>
+                    </div>
+                </div>
+                <div class="side-video-list">
+                    <a href="" class="small-thumbnail">
+                    <img src="/images/thumbnail.png" alt="side-image">
+                    </a>
+                    <div class="vid-info">
+                        <a href="">Web development videos</a>
+                        <p>Easy Tutorial</p>
+                        <p>10k views</p>
+                    </div>
+                </div>
+                <div class="side-video-list">
+                    <a href="" class="small-thumbnail">
+                    <img src="/images/thumbnail.png" alt="side-image">
```

Comment:
This is a lovely Markup,
I think you should try using ID to style unique elements, while you use classes for more general or repeated classes